Spatial_sorting/include/CGAL/hilbert_sort.h

```diff
@@ -43,10 +43,10 @@ namespace internal {
         boost::rand48 random;
         boost::random_number_generator<boost::rand48, Diff_t> rng(random);
         CGAL::cpp98::random_shuffle(begin,end, rng);
-	(Hilbert_sort_2<Kernel, Policy> (k))(begin, end);
+	(Hilbert_sort_2<Kernel, Policy, ConcurrencyTag> (k))(begin, end);
     }
     
-    template <class RandomAccessIterator, class Kernel, class Policy>
+    template <class ConcurrencyTag = Sequential_tag, class RandomAccessIterator, class Kernel, class Policy>
```

Comment:
That cannot compile: default template arguments can only be at the end of the list.

Reply:
@lrineau Are you thinking of classes? This is fine for functions.

Reply:
Yes, sorry.I was reading on my phone and did not see it was function templates.